src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/javadoc/WriteTagCheck.xml

```diff
@@ -8,6 +8,7 @@
  Requires user defined Javadoc tag to be present in Javadoc comment with defined format.
  To define the format for a tag, set property tagFormat to a regular expression.
  Property tagSeverity is used for severity of events when the tag exists.
+ No violation reported in case there is no javadoc.
```

Comment:
Please add to examples some method without javadoc, to make it very clear that there is no validation happening.

@aayushRedHat , it looks like this update is kind of breaking changes in behavior. Do you see what Checks should be used in addition to let users get behavior how it was before?